data_validation/config_manager.py

```diff
@@ -677,7 +677,17 @@ def decimal_too_big_for_64bit(
         casefold_target_columns = {x.casefold(): str(x) for x in target_table.columns}
 
         if arg_value:
-            arg_value = [x.casefold() for x in arg_value]
+            if exclude_cols:
```

Comment:
I suspect this is in the wrong place. I think, but happy to be proved wrong, that `arg_value` is the column CSV passed in by the user, if the user specifies `'*'` then I think `arg_value` is `None`.

If I am right then I think your change only excludes column from a user provided list and not from `'*'`?